e2e/tests/transfer/base_test.go

```diff
@@ -43,6 +43,9 @@ func (s *TransferTestSuite) TestMsgTransfer_Succeeds_Nonincentivized() {
 	t := s.T()
 	ctx := context.TODO()
 
+	// TEST
+	s.Require().True(false)
```

Comment:
Testing the discussion [here](https://github.com/cosmos/ibc-go/pull/3818#discussion_r1226639511)

Reply:
requires renaming the fork e2e before trying again, looks like there is a naming collision, so even though `e2e` fails on native repo, it skips on fork which happens first